nixos/modules/services/home-automation/wyoming/faster-whisper.nix

```diff
@@ -119,6 +119,8 @@ in
         wantedBy = [
           "multi-user.target"
         ];
+        # https://github.com/rhasspy/wyoming-faster-whisper/issues/27
+        environment."HF_HUB_CACHE" = "/tmp";
```

Comment:
In that case, please set `serviceConfig.PrivateTemp = true` as well.

Reply:
Good point. I thought of that and it is actually set automatically when DynamicUser is true. I can still add it though just to avoid other readers to wonder about it. What do you think?

https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#DynamicUser=

> If DynamicUser= is enabled, RemoveIPC= and PrivateTmp= are implied (and cannot be turned off).

Reply:
Good enough for me. Thanks for digging that up.